fcos/release-checklist.md

```diff
@@ -73,6 +73,30 @@ At this point, Cincinnati will see the new release on its next refresh and creat
 ## Refresh metadata (stream and updates)
 
 - [ ] Wait for all releases that will be released simultaneously to reach this step in the process
+
+<details>
+<summary>Rollout general guidelines</summary>
+
+When setting a "Rollout start time" ask "when would be the best time to react to 
+any errors or regressions from updates?"
+Commonly we select 2PM UTC so that the rollout's start at 10am EST, but these can
+be fluid and adjust after talking with the fedora-coreos IRC.
+
+Additionally this is impacted by the day of the week or holidays, again ask 
+"when would be the best time to react to any errors or regressions from updates?"
+
+Commonly we try and release by Tuesday with 48h rollout timelines to insure
+updates are done before the weekend. Consequently the later in the week the release
+the more likely the rollout time allocation will need to shrink or the release will
+need to be pushed. 
+
+A final consideration is a release rebase, i.e. f37 to f38, when this happens its best
+to follow the common case.
+|Day of the week| Rollout Start Time | Time allocation |
+| -------- | ------- | ------- |
+| Tuesday | 2PM UTC | 48H |
+</details>
+
 - [ ] Go to the [rollout workflow](https://github.com/coreos/fedora-coreos-streams/actions/workflows/rollout.yml), click "Run workflow", and fill out the form
```

Comment:
I think it makes sense to put this above the details section.

Reply:
Eh, I tried it both ways and I think I preffer it hidden, it gets cluttered fast.

Reply:
Ah I think that makes sense.